tests/integration/events.test.js

```diff
@@ -370,7 +405,6 @@ describe('GET /events/working-stats', () => {
       });
 
       expect(response.statusCode).toEqual(200);
```

Comment:
Pour la description de ce test, ce serait plutot should return working stats for auxiliary

Reply:
done